apidoc/Titanium/Android/Intent.yml

```diff
@@ -197,17 +296,17 @@ examples:
             });
             intent.addCategory(Ti.Android.CATEGORY_LAUNCHER);
             Ti.Android.currentActivity.startActivity(intent);
-        
-  - title: Create an Intent to get a Contact URI 
+
+  - title: Create an Intent to get a Contact URI
     example: |
         This example creates an intent to retrieve contact information from the user's
-        contacts. 
+        contacts.
 
             var intent = Ti.Android.createIntent({
                 action: Ti.Android.ACTION_GET_CONTENT,
                 type: "vnd.android.cursor.item/phone"
             });
-    
+
   - title: Pick a Photo from the Photo Gallery
     example: |
         This example creates an intent to pick an image from the photo gallery.
```

Comment:
I think we need a blank line between the description and the code. The description is getting rendered as part of the code.